### PR TITLE
{lib,lang}[foss/2017b] Boost-1.65.1-Python-3.6.3, PyCUDA-2017.1.1-CUDA-9.0.176-Python-3.6.3

### PR DESCRIPTION
**Pull request by casparvl**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/Boost/Boost-1.65.1-foss-2017b-Python-3.6.3.eb [hunk 1]
```diff
@@ -1,19 +1,27 @@
+## Author: Caspar van Leeuwen
+## Institute: SURFsara
+## E-mail: caspar.vanleeuwen@surfsara.nl
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/b/Boost/Boost-1.65.1-foss-2017b-Python-3.6.3.eb]: from style check: too many leading '#' for block comment

File: easybuild/easyconfigs/b/Boost/Boost-1.65.1-foss-2017b-Python-3.6.3.eb [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+
+# Fixes this ./boost/python/detail/wrap_python.hpp:50:23: fatal error: pyconfig.h: No such file or directory
+# See also https://svn.boost.org/trac10/ticket/11120
+patches = ['Boost_python_binding.patch']
+
 checksums = [
-    'a13de2c8fbad635e6ba9c8f8714a0e6b4264b60a29b964b940a22554705b6b60',  # boost_1_65_1.tar.gz
-    '5585f98fb67425ec465dcbe2878af046ccd2729a8cdb802acf5d71cfa4022c26',  # Boost-1.65.1_fix-Python3.patch
+    'a13de2c8fbad635e6ba9c8f8714a0e6b4264b60a29b964b940a22554705b6b60', # Boost
+    '83664870ea329feb2fb1e06ec01ab3f9876d9f467a34b0f01089e611e6d694c8' # Patch
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/b/Boost/Boost-1.65.1-foss-2017b-Python-3.6.3.eb]: at least two spaces before inline comment

File: easybuild/easyconfigs/b/Boost/Boost-1.65.1-foss-2017b-Python-3.6.3.eb [hunk 3]
```diff
@@ -25,4 +33,9 @@ dependencies = [
 # also build boost_mpi
 boost_mpi = True
 
+# Compile Boost with -std=c++11, see https://gist.github.com/dennycd/5890475 and https://groups.google.com/forum/#!topic/fenics-support/yrseUpq3PSA
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/b/Boost/Boost-1.65.1-foss-2017b-Python-3.6.3.eb]: line too long

**Comment by migueldiascosta** [comment]
@casparvl I think this PR is now deleting the existing `Boost-1.65.1-foss-2017b-Python-3.6.3.eb` from the repo, can you update the PR by adding the existing easyconfig?

Additionally, please add `download_dep_fail` and `use_pip` to the `PyCUDA` easyconfig and change the order of `Python` and `CUDA` in the `versionsuffix`, to match the existing `PyCUDA-2018.1-intel-2018a-Python-3.6.4-CUDA-9.1.85.eb`?

**Comment by boegel** [comment]
At this point it's probably significantly easier to open a new PR for the `PyCUDA` easyconfig.

In addition, I'm not sure this is actually still worth the trouble, since a newer `PyCUDA` version for a more recent toolchain has already been merged via #6846.

So, I'll go ahead and close this, do let us know if you disagree @casparvl!

**Pull request closed without merging**